tests/unit/mixins/sl-tooltip-enabled-test.js

```diff
@@ -1,54 +1,57 @@
 import Ember from 'ember';
 import mixinUnderTest from 'sl-ember-components/mixins/sl-tooltip-enabled';
+import { test } from 'ember-qunit';
```

Comment:
Was this added to conform to Ember CLI blueprints?  If so it does not match https://github.com/ember-cli/ember-cli/blob/master/blueprints/mixin-test/files/tests/unit/mixins/__name__-test.js